### DISCUSSION TITLE
Welcome to puik Discussions!

### DISCUSSION THREAD

**Discussion by FrancoisQtr** [PrestaShopCorp/puik#273] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by denys202** [comment]
hello @mattgoud could you reopen access to the https://uikit.prestashop.com/ ? The access is restricted from some moment ?
Thank you in advance.
